tests/net/tcp/src/main.c

```diff
@@ -126,6 +126,7 @@ static void handle_server_test(sa_family_t af, struct tcphdr *th);
 static void handle_syn_resend(void);
```

Comment:
add in the commit message that you also fixed the th_seq endian conversion.

Reply:
Done